test/presenters/trial.js

```diff
@@ -56,6 +56,69 @@ describe('trial presenter', () => {
     });
   });
 
+  describe('records', () => {
+    it('sorts records alphabetically by source_id', () => {
+      const trialAttributes = {
+        records: [
+          {
+            "id": "314912a3-60e5-4056-9ae4-89a37e369ae8",
+            "source_id": "ictrp",
+            "url": "http://localhost:10010/v1/trials/ee6883be-3414-456c-82aa-91d802a83c98/records/314912a3-60e5-4056-9ae4-89a37e369ae8",
+            "source_url": "http://apps.who.int/trialsearch/Trial3.aspx?trialid=EUCTR2010-018661-35-BG",
+            "updated_at": "2016-03-25T06:03:02.677Z"
+          },
+          {
+            "id": "401ef4ea-0dca-4af3-b052-84112b2df8fe",
+            "source_id": "euctr",
+            "url": "http://localhost:10010/v1/trials/ee6883be-3414-456c-82aa-91d802a83c98/records/401ef4ea-0dca-4af3-b052-84112b2df8fe",
+            "source_url": "https://www.clinicaltrialsregister.eu/ctr-search/trial/2010-018662-23/FI",
+            "updated_at": "2016-05-25T22:45:53.280Z"
+          },
+          {
+            "id": "91d1e3eb-e512-47ca-88b9-bb1013e55ef0",
+            "source_id": "nct",
+            "url": "http://localhost:10010/v1/trials/ee6883be-3414-456c-82aa-91d802a83c98/records/91d1e3eb-e512-47ca-88b9-bb1013e55ef0",
+            "source_url": "https://clinicaltrials.gov/ct2/show/NCT01193257",
+            "updated_at": "2016-02-07T03:26:01.242Z"
+          },
+        ]
+      };
+      const trial = trialPresenter(trialAttributes);
+
+      should(trial.records).deepEqual([
```

Comment:
When testing, a good practice is to assert a single thing per test. Some people say that a test should actually have a single `assert`, and others say that a test should test a single `concept` (that can have multiple asserts). I'm in the latter team. In this test, you're checking if the records are ordered by `source_id`. To do that, the only thing you need is to check the `source_id`s of the records.

This makes it easier to follow another testing best practice: there should be a single reason for a test to fail. The way it's written now, the test can fail if we, for example, parse the `updated_at` date as a `datetime` object. This has nothing to do with the sorting, which is what you're testing here.